doc/mgr/dashboard.rst

```diff
@@ -44,7 +44,8 @@ The dashboard provides the following features:
   :ref:`dashboard-user-role-management` for details.
 * **Single Sign-On (SSO)**: The dashboard supports authentication
   via an external identity provider using the SAML 2.0 protocol. See
-  :ref:`dashboard-sso-support` for details.
+  :ref:`dashboard-saml2-sso-support` for details. Or using the OAuth2 protocol.
+  See: :ref:`dashboard-oauth2-sso-support` for details.
 * **SSL/TLS support**: All HTTP communication between the web browser and the
```

Comment:
**Single Sign-On (SSO)**: The dashboard supports authentication
  via an external identity provider using the SAML 2.0 protocol and OAuth2 protocol. See
  :ref:`dashboard-saml2-sso-support`  and :ref:`dashboard-oauth2-sso-support` for details.

Reply:
If this change hasn't been made by the end of 02 Dec 2024 UTC, I will make this change and force-push it.